samuroi/util/mask_generator/basic_image_processing.py

```diff
@@ -0,0 +1,33 @@
+__author__ = 'stephenlenzi'
+
+import numpy as np
```

Comment:
these are used in the jupyter notebooks that I will add. I will move the functions there instead and remove the file